packages/components/src/tools-panel/test/index.js

```diff
@@ -172,16 +172,17 @@ const getMenuButton = () => {
  * @return {HTMLElement} The menuButton.
  */
 const openDropdownMenu = async () => {
+	const user = userEvent.setup();
```

Comment:
Do you think it's fine to setup `userEvent` inside a utility function, rather than once per test?

Reply:
Yes, I believe it's fine because a subsequent [`userEvent().setup()`](https://github.com/testing-library/user-event/blob/e93a5af30a2afd05e49abcb37db3f2f212fe0df7/src/setup/setup.ts#L82-L91) will reuse all from the prior call:

* [**Preparing the document**](https://github.com/testing-library/user-event/blob/main/src/document/prepareDocument.ts#L18) - basically adding some event listeners - it's done only once, and if `prepareDocument()` is called, event listeners will not be recreated.
* [**Attaching clipboard stub to window**](https://github.com/testing-library/user-event/blob/e93a5af30a2afd05e49abcb37db3f2f212fe0df7/src/utils/dataTransfer/Clipboard.ts#L113) - will reuse the same stub if it has already been created.
* **[Retrieving the document](https://github.com/testing-library/user-event/blob/e93a5af30a2afd05e49abcb37db3f2f212fe0df7/src/setup/setup.ts#L25)** - will reuse the same document object.

The only thing it won't reuse is the actually returned config object reference, which we never use directly anyway, so we're good to go.